src/UnitTests/test_quote_prov.cpp

```diff
@@ -129,19 +225,20 @@ static void GetCrlTest()
 {
     TEST_START();
 
-    static constexpr uint8_t TEST_FMSPC[] = { 0x00, 0x90, 0x6E, 0xA1, 0x00, 0x00 };
+    static constexpr uint8_t TEST_FMSPC[] = {
+        0x00, 0x90, 0x6E, 0xA1, 0x00, 0x00};
 
     // This is the CRL DP used by Intel for leaf certs
-    static const char* TEST_CRL_URL = "https://api.trustedservices.intel.com/sgx/certification/v1/pckcrl?ca=processor";
+    static const char* TEST_CRL_URL =
+        "https://api.trustedservices.intel.com/sgx/certification/v1/"
+        "pckcrl?ca=processor";

```

Comment:
Why is this split across multiple lines?

Reply:
I don't remember consciously doing this. I believe lots of these are also done by visual studio when they automatically pretty up code